app/konnect/dev-portal/access-and-approval/manage-devs.md

```diff
@@ -3,94 +3,53 @@ title: Manage Developer Access
 no_version: true
 content-type: reference
 ---
-{% konnect_icon connections %} **Connections** manages Dev Portal registration requests and developer accounts.
-The **Connections** section contains three pages to help you manage different aspects of the Dev Portal:
-* The **Requests** page has options to manage developer and application registration requests.
-* The **Developers** page has options to manage access to your Dev Portal for individual developers.
+
+Within the {% konnect_icon dev-portal %} **Dev Portal** section there are three ways to manage access and authorization settings for the Dev Portal: 
+* The **Access Requests** page lists developer registration and application registration requests.
+* The **Developers** page list of developers who have requested access to the Dev Portal, along with the developer's current status and associated applications.
 * The **Applications** page lists applications and their statuses.
 
 To allow automatic approval of developer registration requests,
 enable [auto approve](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps).
 
-## Requests page
+## Access Requests page
 
-The **Requests** page is available by clicking {% konnect_icon connections %} **Connections**, from the {{site.konnect_short_name}} navigation menu. On the **Requests** page, you can manage registration requests for developers by clicking the **Developers** tab, and application requests by clicking the **Applications** tab.
+The **Access Requests** page is available by clicking {% konnect_icon dev-portal %} **Dev Portal**, from the {{site.konnect_short_name}} navigation menu. On the **Access Requests** page, you can manage registration requests for developers using the **Developers** tab, and application requests using the **Applications** tab. There are three options for managing incoming requests: **Approve**, **Reject**, and **Delete**. When auto-approve is disabled all registration requests must be manually approved from the **Access Requests** page. 
```

Comment:
```suggestion
The **Access Requests** page is available by clicking {% konnect_icon dev-portal %} **Dev Portal**, from the {{site.konnect_short_name}} navigation menu. On the **Access Requests** page, you can manage registration requests for developers using the **Developers** tab, and application requests using the **Applications** tab. When auto-approve is disabled, all registration requests must be manually approved from the **Access Requests** page. 
```
I think this content could be cut down a little bit because this is explained in the task sections that follow this one.